src/app.js

```diff
@@ -46,6 +47,9 @@ app.get("/api/hc", (req, res) => {
 });
 app.use("/api", routes);
 
+app.use(adminRouterPath, isAdmin, adminRouter);
+console.log(`AdminJS started on http://localhost:6001${adminRouterPath}`);
```

Comment:
`/bin/www`에 이런 코드가 있는데요,
```const port = normalizePort (process.env.PORT || '6001');```
이 코드를 참조 하여 아래처럼 포트를 보여주는 것도 좋아보입니다!

```suggestion
console.log(`AdminJS started on http://localhost:${process.env.PORT || '6001'}${adminRouterPath}`);
```